hypothesis-python/docs/schema_observations.json

```diff
@@ -44,6 +44,14 @@
                         "uniqueItems": true
                     }
                 },
+                "timing": {
+                    "type": "object",
+                    "description": "The time in seconds taken by non-overlapping parts of this test case.  Hypothesis reports generate_before_test, execute_test, and generate_during_test.",
```

Comment:
what would be an example of an "overlapping" part of the test case? Is this just intended to convey that the three phases — before, execute, during — are disjoint? I'm left a bit confused (which means that the casual reader may be as well).

Reply:
Yep, intended to express that these are disjoint times such that taking the sum gives the correct total.  (though test execution and generate-during are interleaved, we subtract the latter from our raw measurement to yield the former)